vslib/src/SwitchStateBase.cpp

```diff
@@ -2424,3 +2439,214 @@ void SwitchStateBase::debugSetStats(
         m_countersMap[key][kvp.first] = kvp.second;
     }
 }
+
+sai_status_t SwitchStateBase::initialize_voq_switch_objects(
```

Comment:
this function has mistakes which i pointed in previous pr, some empty lines added, empty liens before if, spaces before //

Reply:
Other option is: I can remove this file form this PR. Once this PR #702 is merged, I can make a commit get the local port id setting for local system port.

Reply:
Removed the changes related to local port oid setting for system ports so that the SwitchStateBase.cpp will not be in this PR. Once the dependent PR #702 and PR #701 are merged the removed changes will be put back through another commit.

Reply:
Added back the changes to set local port attr (local port oid) in corresponding system port info object